monai/data/meta_tensor.py

```diff
@@ -125,19 +126,20 @@ def __init__(
         elif isinstance(x, MetaTensor):
             self.applied_operations = x.applied_operations
         else:
-            self.applied_operations = self.get_default_applied_operations()
+            self.applied_operations = MetaObj.get_default_applied_operations()
 
         # if we are creating a new MetaTensor, then deep copy attributes
         if isinstance(x, torch.Tensor) and not isinstance(x, MetaTensor):
             self.meta = deepcopy(self.meta)
             self.applied_operations = deepcopy(self.applied_operations)
         self.affine = self.affine.to(self.device)
 
-    def _copy_attr(self, attribute: str, input_objs: list[MetaObj], default_fn: Callable, deep_copy: bool) -> None:
-        super()._copy_attr(attribute, input_objs, default_fn, deep_copy)
-        val = getattr(self, attribute)
-        if isinstance(val, torch.Tensor):
-            setattr(self, attribute, val.to(self.device))
+    def _copy_attr(self, attributes: list[str], input_objs, defaults: list, deep_copy: bool) -> None:
+        super()._copy_attr(attributes, input_objs, defaults, deep_copy)
+        for a in attributes:
+            val = getattr(self, a)
+            if isinstance(val, torch.Tensor):
+                setattr(self, a, val.to(self.device))
 
     @staticmethod
     def update_meta(rets: Sequence, func, args, kwargs) -> Sequence:
```

Comment:
may have "strict" and "dev" modes to control manual modifications of the meta information